install.py

```diff
@@ -0,0 +1,115 @@
+import os
+from zipfile import ZipFile
+
+IGUASSU_BRANCH="release/v1.0.0"
+ARREBOL_BRANCH="feature/remote-worker"
+WEB_UI_BRANCH="master"
+
+def unzip_file(file_name):
+    zf = ZipFile(file_name, 'r')
+    zf.extractall()
+    zf.close()
+
+# Download and extract the archive file from repository
+def download_repository(url):
+    os.system("wget {}".format(url))
+    file_name = os.path.basename(url)
+    unzip_file(file_name)
+    os.remove(file_name)
+
+def install_iguassu():
+    print("---> Installing Iguassu")
+    url = "https://github.com/ufcg-lsd/iguassu/archive/{}.zip".format(IGUASSU_BRANCH)
+    download_repository(url)
+
+    iguassu_dir_name = "iguassu-{}".format(IGUASSU_BRANCH.replace("/", "-"))
+    os.chdir(iguassu_dir_name)
+    os.system("mvn clean install -DskipTests")
+    os.chdir("..")
+    return iguassu_dir_name
+
+def install_arrebol():
+    print("---> Installing Arrebol")
+    url = "https://github.com/ufcg-lsd/arrebol/archive/{}.zip".format(ARREBOL_BRANCH)
+    download_repository(url)
+
+    arrebol_dir_name = "arrebol-{}".format(ARREBOL_BRANCH.replace("/", "-"))
+    os.chdir(arrebol_dir_name)
+    os.system("mvn clean install -DskipTests")
+    os.chdir("..")
+    return arrebol_dir_name
+
+def install_web_ui():
+    print("---> Installing Web UI")
+    url = "https://github.com/emanueljoivo/iwi/archive/{}.zip".format(WEB_UI_BRANCH)
+    download_repository(url)
+
+    iwi_dir_name = "iwi-{}".format(WEB_UI_BRANCH.replace("/", "-"))
+    os.chdir(iwi_dir_name)
+    os.system("npm install")
+    os.chdir("..")
+    return iwi_dir_name
+
+oauth_storage_service_token_url="oauth_storage_service_token_url"
```

Comment:
The name of these properties on the .env file is a bit different. Please, verify in your local version the correct properties names for the Web UI.

Reply:
For example, for the client identification, the property has the name`VUE_APP_OWNCLOUD_OAUTH_CLIENT_ID`.

Reply:
Done.